src/llvm/analysis/DefUse.cpp

```diff
@@ -327,8 +342,10 @@ void LLVMDefUseAnalysis::addDataDependence(LLVMNode *node, PSNode *pts,
             if (!GV || !GV->hasInitializer()) {
                 static std::set<const llvm::Value *> reported;
                 if (reported.insert(llvmVal).second) {
-                    llvm::errs() << "No reaching definition for: " << *llvmVal
-                                 << " off: " << *ptr.offset << "\n";
+                    llvm::errs() << "No reaching definition for: " << *llvmVal;
+                    if (mem->getUserData<llvm::Value>())
+                        llvm::errs() << " in: " << getInstName(mem->getUserData<llvm::Value>());
```

Comment:
I think that using `getInstName` is redundant here, using just the << operator should be working fine, right? Something like:
```C++
const llvm::Value *memVal = mem->getUserData<llvm::Value>();
if (memVal)
    llvm::errs() << " in: " << *memVal;
```